### ISSUE TITLE
Line chart with date values in XAxis not rendering chart clearly

### ISSUE THREAD

**Issue by durgatjb** [PhilJay/MPAndroidChart#2857]
Sample dataset: For the below dataset getting a small line like (backward slash) as graph. The line chart is not rendered properly. 
 "date_x_axis": [
        "2017-01-11T00:00:00.000Z",
        "2017-01-16T00:00:00.000Z",
        "2017-01-17T00:00:00.000Z",
        "2017-01-21T00:00:00.000Z",
        "2017-01-23T00:00:00.000Z",
        "2017-01-24T00:00:00.000Z",
        "2017-01-05T00:00:00.000Z",
        "2017-01-27T00:00:00.000Z",
        "2017-01-29T00:00:00.000Z",
        "2017-02-03T00:00:00.000Z",
        "2017-02-06T00:00:00.000Z",
        "2017-02-09T00:00:00.000Z"
      ],
      "values_y_axis": [
        0.219999998808,
        0.34999999404,
        0.180000007153,
        42.8400001526,
        0.180000007153,
        21.6299991608,
        28.5599994659,
        21.4200000763,
        0.180000007153,
        0.280000001192,
        0.180000007153,
        0.34999999404
      ]

**Comment by durgatjb** [comment]
Hi is there any workaround for the above issue? I have implemented in my project and need to address this issue immediately

**Comment by LamraniOthmane** [comment]
Hi, i have the same issue, can you tell me if you have found any solution ?

**Comment by durgatjb** [comment]
Yeah the issue got resolved. In my case, the date in X-Axis was not sorted. I sorted the dates and it worked as expected. Let me know if this works to you too! thanks.
